autoPyTorch/pipeline/components/preprocessing/tabular_preprocessing/utils.py

```diff
@@ -0,0 +1,17 @@
+from typing import Any, Dict, List
+
+from sklearn.base import BaseEstimator
+
+
+def get_tabular_preprocessers(X: Dict[str, Any]) -> Dict[str, List[BaseEstimator]]:
```

Comment:
Do you think you can add a description saying:

We expect that the fit dictionary would have previously fitted the numerical/categorical nodes that will build  the column transformer of the tabular preprocessing component. This function parses the fict dictionary and extract such components.